src/test/resources/features/apicuritoTests.feature

```diff
@@ -114,16 +113,18 @@ Feature: simpleTestMainPage
 
   @allGetParameters
   Scenario: fill all get parameters
-    And create a new path with plus sign with name "MyGet"
+    And create a new path "MyGet" with link "false"
```

Comment:
And is not correct here, it only "copies" the previous gherkin step meaning, either don't use and at all or if you do, be strict with it (use it correctly, start with When/Given)

Reply:
Should be correct now.